README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
 <h1>envd</h1>
-<p>Development environment for data science and AI/ML</p>
+<p>Docker alternative for AI/ML applications</p>
```

Comment:
I personally prefer `development environment` over `docker alternative`. 

Reasons:
- When searching about `docker alternative`, the results would by containerd/podman, which is not actually what we're referring to.
- Let's say if a researcher came to this project, he might think "I don't need docker, why even bother with having a docker alternative". The slogan itself should create an "Aha moment", especially for algorithm researchers/scientists who don't know about docker.

Reply:
* I agree that `docker alternative` is ambiguous. It's more like `Dockerfile alternative`.
* I think we also support serving and we will add more serving-related features. So it's not just `development environment`
* We should pick up a slogan for both DS/ML and the infra team.

Reply:
Second to that. We need to come up with a better one, ideally one that reflects what "envd" means and easy for people to remember. Are we settled with "envd" for the project name?

Reply:
I use Docker alternative from this tweet. https://twitter.com/Rami_Krispin/status/1540328916589498368

I think docker alternative may be easy to understand from the perspective of ds and algorithm engineers.

Reply:
How about `Clean and portable workspaces`?

Reply:
I cannot get it from the text. The description is too general for me.

Reply:
> Second to that. We need to come up with a better one, ideally one that reflects what "envd" means and easy for people to remember. Are we settled with "envd" for the project name?

Yep, the name is envd.

Reply:
Let's discuss in https://github.com/tensorchord/envd/discussions/834

Reply:
I reverted the change here.